devtools/maven/src/main/java/io/quarkus/maven/NativeImageMojo.java

```diff
@@ -186,278 +176,164 @@ public NativeImageMojo() {
     @Override
     public void execute() throws MojoExecutionException, MojoFailureException {
 
-        if (project.getPackaging().equals("pom") && appArtifact == null) {
```

Comment:
Was there a reason to remove `appArtifact` check here?
`appArtifact` is used in the platform integration tests. The reason I allowed `pom` project and `appArtifact` combination was that the project was simply pulling all the code from the plugin config and dependencies and didn't produce any JAR.

Reply:
It looks like somebody has removed pom packaging from the platform tests, although the rest of the config remains the same. It could still be just a POM though.